src/features/settings/settingsSlice.ts

```diff
@@ -42,17 +42,14 @@ export const settingsSlice = createSlice({
     setMarketLabelled: (state, action: PayloadAction<MarketLabelled>) => {
       state.marketsLabelled = [...(state.marketsLabelled || []), action.payload];
     },
-    setTdexDaemonBaseUrl: (state, action: PayloadAction<string>) => {
-      state.tdexDaemonBaseUrl = action.payload;
-    },
-    setMacaroonCredentials: (state, action: PayloadAction<string | undefined>) => {
-      state.macaroonCredentials = action.payload;
-    },
     setTdexdConnectUrl: (state, action: PayloadAction<string | undefined>) => {
       state.tdexdConnectUrl = action.payload;
     },
+    setIsInitialized: (state, action: PayloadAction<boolean>) => {
+      state.isInitialized = action.payload;
+    },
     logout: (state) => {
-      state.macaroonCredentials = undefined;
+      state.isInitialized = false;
```

Comment:
what isInitialized means? we can logout (ie. deleting tdexdconnect URL) but the daemon can be still initialized, we only need to pair again.

Reply:
I removed the macaroon from the state because i thought we didn't want to connect directly with the daemon anymore.

Is storing the macaroon in clear in the local storage a good practice tho? Consider that's the admin one and grants access to all daemon's RPCs.

I'm going to make some changes to support both the "direct" and "proxy" connection, but at the moment I don't see any other way rather than storing the macaroon in LS to directly connect to a daemon.

Reply:
I meant that here lougout should not change the `isInitialized` state, unless does not refer to the daemon's state?

Reply:
or you do that because you cleaning?

Reply:
`isInitialized` name is tricky because it first stems from daemon `isReady` state but then doesn't follow daemon state. Maybe should be renamed `isClientLoggedIn` or something like that to avoid confusion.